backend/mr-db-sql/src/main/kotlin/io/featurehub/db/services/PersonSqlApi.kt

```diff
@@ -292,6 +302,49 @@ open class PersonSqlApi @Inject constructor(
     return personToken
   }
 
+  @Transactional
+  override fun createServicePerson(name: String, createdBy: UUID?): CreatedServicePerson? {
+    val created = if (createdBy == null) null else convertUtils.byPerson(createdBy)
+    if (createdBy != null && created == null) {
+      return null
+    }
+
+    val servicePerson = DbPerson()
+    servicePerson.name = name
+    with(servicePerson) {
+      id = UUID.randomUUID()
+      email = "${id}@admin.sa.featurehub.io"
```

Comment:
Should this be configurable instead of hard coded here?

Reply:
@rvowles ^^

Reply:
This is not an email address, its like a Google service account address, it is never emailed and never communicated with. At least that is my plan - otherwise we would require the person provisioning the account to provide it. It is hard coded deliberately...